### PR TITLE
feat: make buttons 100% width gf-232

### PR DESCRIPTION
**Pull request by HoroshkoMykhailo**
Summary:

- Made the Button component 100% width by default.
- Removed the unnecessary properties from button wrappers across the app.
- Everything stayed the same.

image

image

image

image

image

image

image

image

### CODE REVIEW

File: apps/frontend/src/pages/projects/styles.module.css [hunk 1]
```diff
@@ -25,3 +25,7 @@
 	line-height: 150%;
 	color: var(--color-text-secondary);
 }
+
+.button-wrapper {
```
Comment (liza-veis) [inline apps/frontend/src/pages/projects/styles.module.css:29]: not sure we need min-width here, on figma this button just takes the size of it's label + paddings

File: apps/frontend/src/pages/profile/libs/components/edit-user-form/styles.module.css [hunk 2]
```diff
@@ -2,18 +2,17 @@
...
+	align-items: flex-start;
 }
 
 .inputs-wrapper {
 	display: flex;
 	flex-direction: column;
 	gap: 16px;
 	width: 100%;
-	max-width: 500px;
+	min-width: 500px;
```
Comment (liza-veis) [inline apps/frontend/src/pages/profile/libs/components/edit-user-form/styles.module.css:13]: I think we should keep max-width here, the problem why it's not full width is that one of the containers lacks width 100%

File: apps/frontend/src/pages/profile/libs/components/delete-account/styles.module.css [hunk 3]
```diff
@@ -18,3 +18,7 @@
 	font-size: 16px;
 	line-height: 150%;
 }
+
+.button-wrapper {
+	min-width: 190px;
```
Comment (liza-veis) [inline apps/frontend/src/pages/profile/libs/components/delete-account/styles.module.css:23]: the same, on figma this button just takes it's content width, we need to use flex here to align item to one of the sides

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/components/button/styles.module.css [hunk 4]
```diff
@@ -1,12 +1,14 @@
 .button {
+	display: block;
 	align-content: center;
+	width: 100%;
 	min-height: 44px;
 	padding: 10px 16px;
-	font-family: Inter, sans-serif;
 	font-size: 16px;
```
Comment (liza-veis) [inline apps/frontend/src/libs/components/button/styles.module.css:7]: why did you remove font-family here?

**Review by what1s1ove — APPROVED** [review]
lgtm after Lisa's comments
